tests/test_models.py

```diff
@@ -1168,21 +1168,33 @@ def test_channel_code_raises(self):
 
     @mock.patch('integrated_channels.integrated_channel.models.LearnerExporter')
     def test_get_learner_data_exporter(self, mock_learner_exporter):
+        """
+        The configuration returns the appropriate learner exporter,
+        """
         mock_learner_exporter.return_value = 'mock_learner_exporter'
         assert self.config.get_learner_data_exporter(None) == 'mock_learner_exporter'
 
     @mock.patch('integrated_channels.integrated_channel.models.LearnerTransmitter')
     def test_get_learner_data_transmitter_raises(self, mock_learner_transmitter):
+        """
+        The configuration returns the appropriate learner transmitter,
+        """
         mock_learner_transmitter.return_value = 'mock_learner_transmitter'
         assert self.config.get_learner_data_transmitter() == 'mock_learner_transmitter'
 
     @mock.patch('integrated_channels.integrated_channel.models.CourseExporter')
     def test_get_course_data_exporter_raises(self, mock_course_exporter):
+        """
+        The configuration returns the appropriate course exporter,
+        """
         mock_course_exporter.return_value = 'mock_course_exporter'
         assert self.config.get_course_data_exporter(None) == 'mock_course_exporter'
 
     @mock.patch('integrated_channels.integrated_channel.models.CourseTransmitter')
     def test_get_course_data_transmitter_raises(self, mock_course_transmitter):
+        """
+        The configuration returns the appropriate course transmitter,
```

Comment:
@UmanShahzad Typo: "," should be ".", for this test and three tests above.

Reply:
:+1: